flutter-idea/src/io/flutter/FlutterBundle.properties

```
@@ -213,6 +213,8 @@ settings.allow.tests.tooltip=The directory must be marked as a sources root, suc
 settings.font.packages=Font Packages
 settings.show.all.configs=Show all possible run configurations for apps or tests, even if a created configuration already exists
 settings.show.all.configs.tooltip=If there is a defined run configuration to watch a specific test and one to just run it, show both.
+settings.jcef.browser=Use JCEF browser to show embedded DevTools.
```

I don't see other fields having the ending, so I'm suggesting this.

```suggestion
settings.jcef.browser=Use JCEF browser to show embedded DevTools
```